Sources/AST/ASTPass/ASTPassContext.swift

```diff
@@ -100,6 +100,13 @@ extension ASTPassContext {
     set { self[EnumDeclarationContextEntry.self] = newValue }
   }
 
+  /// Contextual information used when visiting declarations in an event declaration, such as the
+  /// name of the event the members belong to.
```

Comment:
... trait declaration, ... name of the trait

Reply:
yes...